Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -1503,6 +1562,53 @@ end Point
 
 end Affine
 
+section Map
+
+variable {S : Type*} [CommRing S] (f : R →+* S) (P Q : Fin 3 → R)
+
+protected lemma map_smul (u : R) : f ∘ (u • P) = f u • (f ∘ P) := by
+  ext i; fin_cases i <;> simp [smul_fin3]
```

Comment:
Move this to below `comp_fin3` too? Since this is a general fact about `Fin 3`s.